app/models/course/survey/response.rb

```diff
@@ -18,7 +18,8 @@ def submitted?
   def submit
     self.submitted_at = Time.zone.now
     self.points_awarded = survey.base_exp
-    self.points_awarded += survey.time_bonus_exp if submitted_at <= survey.end_at
+    self.points_awarded += survey.time_bonus_exp if \
```

Comment:
So if `allow_response_after_end` is not checked, time_bonus exp won't be awarded ? is that correct ?